cfme/tests/containers/test_provider_configuration_menu.py

```diff
@@ -52,4 +53,7 @@ def test_remove_selected_containers_provider():
     name = select_first_provider_and_get_its_name()
     ContainersProvider(name).delete(cancel=False)
     sel.refresh()
-    assert not Quadicon(name).exists
+    assert wait_for(lambda *args: not Quadicon(name).exists,
+                    'Waiting for containers provider to be removed.',
+                    delay=5,
+                    num_sec=60.0), 'Failed to remove containers provider'
```

Comment:
missing fail_func that would call selenium's refresh or the reload button click.